data/json/items/ammo/12mm.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "12mm",
+    "type": "AMMO",
+    "name": "H&K 12mm",
+    "name_plural": "H&K 12mm",
+    "description": "The Heckler & Koch 12mm projectiles are used in H&K railguns.  It's made of a ferromagnetic metal, probably cobalt.",
+    "weight": 11,
+    "volume": 1,
+    "price": 50000,
+    "material": [ "steel" ],
+    "symbol": "=",
+    "color": "blue",
+    "count": 20,
+    "stack_size": 40,
+    "ammo_type": "12mm",
+    "range": 70,
+    "damage": 50,
+    "pierce": 40,
+    "dispersion": 45,
+    "recoil": 105
```

Comment:
> There's nothing wrong with it being 105.

It's three significant figures which implies a greater degree of precision than is actually present - that last 0.5% makes no difference. It's the same reason we don't allow a dispersion of `45.6`.

So 110 would be equally reasonable but 103 would not.

Lots of little differences also make comparisons for equality and statistical binning more tricky but that's a side issue.

A lot of the aiming and recoil updates involve some fairly hefty math so we should avoid making the values more awkward then necessary for no tangible gain